src/license.rs

```diff
@@ -9,6 +9,7 @@
  */
 
 use md5::Context;
+use path_dedot::*;
```

Comment:
start imports are frowned upon. afaict you can just import the trait

Reply:
I was thinking the same, got it from the docs :wink: